mypy.ini

```diff
@@ -9,3 +9,12 @@ allow_untyped_decorators = True
 
 [mypy-etest_test.system_test]
 allow_untyped_decorators = True
+<<<<<<< HEAD
```

Comment:
Conflict markers.  pre-commit should have a check for this but if not we can add it later.

Reply:
Its a bit concerning that no step in the testing chain caught this.

Reply:
I agree.  Feel free to create a pull request adding another pre-commit as another pull request and rebasing this work on that.